docs/tokens.md

```diff
@@ -66,13 +62,21 @@ TO DO
 ## Wallet
 
 TO DO
-## Issue and Transfer Tokens
+## Issuing and Transferring Tokens
 
 **TODO who are these examples meant for?**
+**TODO It looks like alice is giving carl normal tokens?**
```

Comment:
What do you mean by normal tokens? MLS-01 is the "basic token" standard on ML. If you issued a RoyToken it would likely be MLS-01, if you think of a basic token on Ethereum the chances are it's an erc-20 token, this is the closest thing to that on ML. We also have MLS-02 (well not yet we don't - they're confidential tokens) and MLS-03 which are NFTs. Given the code below is for MLS-01 I think it's fair to call them "normal tokens" in the sense they don't have any special properties for CTs or being NF. I'd stick to the MLS-01 nomenclature over "normal" though as changing the wording everywhere will get confusing.

Reply:
This was basically me confused and mumbling to myself. I meant coins. I didn't understand the TxOutput containing both a nonzero Value and a TokenIssuance data field.After talking with Sam I understood that you can transfer tokens and coins in the same transaction.